Resources/Prototypes/Entities/Structures/plastic_flaps.yml

```diff
@@ -0,0 +1,65 @@
+- type: entity
+  id: PlasticFlapsClear
+  parent: BaseStructureDynamic
+  name: airtight plastic flaps
+  suffix: Clear
+  description: Heavy duty, airtight, plastic flaps. Definitely can't get past those. No way.
+  placement:
+    mode: SnapgridCenter
+  components:
+  - type: Sprite
+    netsync: false
+    sprite: Structures/plastic_flaps.rsi
+    state: plasticflaps
+  - type: Physics
+  - type: Fixtures
+    fixtures:
+    - shape:
+        !type:PhysShapeAabb
+        bounds: "-0.49,-0.49,0.49,0.49"
+      mass: 100
+      mask:
+      - Passable
+      layer:
+      - MobImpassable
+  - type: Airtight
+    fixVacuum: true
```

Comment:
should these really be airtight

Reply:
It's how they were in TG, I'm just mainly copying verbatim.

Don't know if it's planned future use is with uses with Mule deliveries as well but it does keep from worrying about those backdoor areas getting spaced if there was a hallway explosion or something.

Reply:
![image](https://user-images.githubusercontent.com/47410468/153141733-684d64da-6625-42c2-b895-f1c524071071.png)

removed airtight if that will get it through, if a mapper needs one, it can be made later.
Just as well forgot to have anchor on it so should be good now.

Reply:
usually they also have windoors behind them to prevent unwanted guests, which would also seal them. another option could just be an airtight variant yeah